serverless/discover/about-serverless.adoc

```diff
@@ -44,5 +44,5 @@ ifdef::openshift-enterprise[]
 * xref:../../operators/understanding/crds/crd-extending-api-with-crds.adoc#crd-extending-api-with-crds[Extending the Kubernetes API with custom resource definitions]
 * xref:../../operators/understanding/crds/crd-managing-resources-from-crds.adoc#crd-managing-resources-from-crds[Managing resources from custom resource definitions]
 endif::[]
-// above links only included for OCP currently until CRD docs are available in OSD
+// above links only included for OCP currently until CRD docs are available in OSD and ROSA
```

Comment:
Has it been checked if these docs will be added to ROSA eventually? Not sure if they use Operators too or just managed services?

Reply:
ROSA indeed uses Operators the same way as OCP does.